optashift-employee-rostering-gwtui/pom.xml

```diff
@@ -102,6 +102,27 @@
           <moduleName>org.optaplanner.openshift.employeerostering.gwtui.WorkerRosteringWebapp</moduleName>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>2.3</version>
```

Comment:
optashift-employee-rostering-gwtui needs to use the same plugin versions as the rest of the build.
So maven plugin versions should be set (if needed at all) in the parent pom, so in /pom.xml instead of the child pom.

I think we want the parent pom to extend the jboss parent pom so we don't need to set up the vanilla plugin versions:

```
  <parent>
    <groupId>org.jboss</groupId>
    <artifactId>jboss-parent</artifactId>
    <version>22</version>
  </parent>
```

(check if 22 is the latest)